pkg/webhooks/resourcebinding/mutating/mutating.go

```diff
@@ -77,7 +78,7 @@ func ResourceBindings(ar admissionv1.AdmissionReview) *admissionv1.AdmissionResp
 	}
 
 	// Check if its workload, skip suspend if not.
-	isWorkload, err := utils.IsWorkload(rb.Spec.Resource)
+	isWorkload, _, err := workload.TryGetNewWorkloadFunc(rb.Spec.Resource)
```

Comment:
Should we depend on this condition to check whether rb reference to a workload? In Karmada, based on rb.spec.Replicas>0 to determine whether it is a workload, but now this judgment is based on whether the workload has been registered, if there is no registration of workload judgment result is wrong.